hoomd/hpmc/integrate.py

```diff
@@ -311,12 +316,13 @@ class Sphere(_HPMCIntegrator):
     Args:
         seed (int): Random number seed
         d (float): Maximum move displacement, Scalar to set for all types, or a
-            dict containing {type:size} to set by type.
+          dict containing {type:size} to set by type.
         a (float): Maximum rotation move to set for all types, or a dict
-            containing {type:size} to set by type.
+          containing {type:size} to set by type.
         move_ratio (float, only used with **orientable=True**): Ratio of
-            translation moves to rotation moves. (added in version 2.3)
+          translation moves to rotation moves. (added in version 2.3)
```

Comment:
I don't know if we need added in version 2.3 anymore since we are releasing this as 3.0.

Reply:
Removed 1fc0c52f6